lib/expr.c

```diff
@@ -3947,6 +3972,14 @@ parse_near_options(efs_info *q,
     }
   }
 
+  if (end < q->str_end && end[0] == ',') {
+    const char *min_interval_start = end + 1;
+    data->options.near.min_interval =
+      grn_atoi(min_interval_start,
+               q->str_end,
+               &end);
+  }
```

Comment:
We should set the default value of `data->options.near.min_interval` in this function when this block isn't used.

Reply:
Fixed.